lib/debug/debugger_ui.dart

```diff
@@ -678,7 +680,8 @@ class LibrariesTab extends MTab {
     });
   }
 
-  void _render(ObservatoryLibrary lib, CoreElement element) {
+  void _render(dynamic _lib, CoreElement element) {
+    ObservatoryLibrary lib = _lib;
```

Comment:
My mistake, but all ObservatoryLibrary here should be DebugLibrary, can you fix?

Reply:
sure!